<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-NUE1PJR8" ignored="0" name="UserTest" tests="3" failures="1" timestamp="14 Sep 2022 12:16:24 GMT" time="3.016" errors="0">
  <testcase name="countOfMaleUsers" time="0.467" classname="userpgtest.UserPgTest"/>
  <testcase name="varifyMobileNoOfUsers" time="2.394" classname="userpgtest.UserPgTest">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
at userpgtest.UserPgTest.varifyMobileNoOfUsers(UserPgTest.java:27)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- varifyMobileNoOfUsers -->
  <testcase name="veriryOperatorsPgLoad" time="0.149" classname="userpgtest.UserPgTest"/>
</testsuite> <!-- UserTest -->
